app/forms/schools/year2020_form.rb

```diff
@@ -57,12 +95,14 @@ def save!
         school_cohort.core_induction_programme = core_induction_programme
         school_cohort.save!
 
-        EarlyCareerTeachers::Create.call(
-          full_name: full_name,
-          email: email,
-          school_cohort: school_cohort,
-          mentor_profile_id: nil,
-        )
+        get_participants.each do |participant|
+          EarlyCareerTeachers::Create.call(
```

Comment:
Do we test that this is called the right number of times with the correct args?

Reply:
Hmm, `correct number of times` - probably yes, we have a test saying 'if form has 3 participants, and you save, then the school cohort should have 3 participants'. Close enough.

But they could in principle have the wrong details. I will add a test.